xblock/test/django/test_request.py

```diff
@@ -80,12 +80,12 @@ def test_headers(self):
     def test_content_type(self):
         # Default charset
         self.assertEquals(
-            self._as_django(content_type='application/json')['Content-Type'],
-            'application/json; charset=UTF-8'
+            self._as_django(content_type='text/plain')['Content-Type'],
+            'text/plain; charset=UTF-8'
```

Comment:
We don't make Django responses that are plain text, we make JSON responses, so I think it's important to keep the same content_type.